server/service/translator.go

```diff
@@ -88,7 +95,15 @@ func (svc *Service) Translate(ctx context.Context, payloads []fleet.TranslatePay
 			}
 			translateFunc = translateHostToID
 		default:
-			return nil, fleet.NewErrorf(fleet.ErrNoUnknownTranslate, "Type %s is unknown.", payload.Type)
+			// if no supported payload type, this is bad regardless of authorization
+			svc.authz.SkipAuthorization(ctx)
+			return nil, badRequestErr(
+				fmt.Sprintf("Type %s is unknown. ", payload.Type),
+				fleet.NewErrorf(
+					fleet.ErrNoUnknownTranslate,
+					"Type %s is unknown.",
+					payload.Type),
```

Comment:
Does this internal error provide any value? The error message is mostly duplicated.

Reply:
It's not clear how `fleet.ErrNoUnknownTranslate` is meant to be used, but it resembles other such error `int`s that are used elsewhere, so figured might as well keep it around in the internal error since it was present before.